src/index.ts

```diff
@@ -10,9 +10,12 @@ import { beginWatchingWorkloads } from './supervisor/watchers';
 import { loadAndSendWorkloadEventsPolicy } from './common/policy';
 import { sendClusterMetadata } from './transmitter';
 import { setSnykMonitorAgentId } from './supervisor/agent';
-import { scrapeData } from './data-scraper';
+import { scrapeData, scrapeDataV1 } from './data-scraper';
 import { setupHealthCheck } from './healthcheck';
 
+export const sysdigV1 = 'V1';
+export const sysdigV2 = 'V2';
+
```

Comment:
these consts are also being exported from `src/healthcheck.ts`, I think we only need to export from one place

Reply:
ah, got some circular import issues, and then missed to unexport them - redoing.